specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/LongTermRetentionPolicies.json

```diff
@@ -243,6 +220,33 @@
           "readOnly": true
         }
       }
+    },
+    "LongTermRetentionPolicyProperties": {
+      "description": "Properties of a long term retention policy",
+      "type": "object",
+      "properties": {
+        "makeBackupsImmutable": {
+          "description": "The setting whether to make LTR backups immutable",
+          "type": "boolean"
+        },
+        "weeklyRetention": {
+          "description": "The weekly retention policy for an LTR backup in an ISO 8601 format.",
+          "type": "string"
+        },
+        "monthlyRetention": {
```

Comment:
>monthlyRetention [](http://example.com/codeflow?start=9&length=16)

consider defining "format": "date-time"

Reply:
monthlyRetention is in the form of PxY where x is the number of months